### PR TITLE
Add items to the unbundled module to fix webpacking

### PR DESCRIPTION
**Pull request by gcampbell-msft**
This is a workaround to make sure the web-packing happens correctly so that the extension can be functional. 
The items added to the "unbundledModule" are packages that are used from the node-serialport package, and this ensures that they are included and not bundled. 

I have created a package locally using 'vsce package' in order to create a vsix and tested that the commands seem to work. I didn't test each command end to end, but commands exist and are able to be called.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function getEntry() {
   const mod = JSON.parse(npmListRes);
   const unbundledModule = ['impor', 'uuid',
   // usb-native modules can not be bundled
-  'usb-detection', '@serialport', 'bindings', 'serialport'];
+  'usb-detection', '@serialport', 'bindings', 'serialport', 'debug', 'ms', 'node-gyp-build', 'node-addon-api'];
```
Comment (benmcmorran) [inline webpack.config.js:15]: It's weird that `node-gyp-build` is required in a production build. I'm pretty sure that package is used at build time to create native binaries, but I don't see why there would be a runtime dependency. That said, if it unblocks the release, I think we can keep it for now. Can you add a comment explaining why these packages were added?

File: package-lock.json [hunk 2]
```diff
@@ -16678,7 +16678,7 @@
       "integrity": "sha512-m62ShEObQ39CfralilEQRjH6oAMtNCV1xJyEx5LpRYUVN+EviphDgUc/F3hnYbADmkiNs67Y+3ylmlG7Lnu+FA==",
       "dev": true,
       "requires": {
-        "es5-ext": "0.10.53",
```
Comment (benmcmorran) [inline package-lock.json:16681]: Normally this would be fine, but I'm hesitant to change `package-lock.json` between release candidates. Can you revert the changes to this file and check if we still have a passing build? I don't see why changes to the webpack configuration would require `package-lock.json` changes.

**Review by benmcmorran — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (gcampbell-msft) [reply webpack.config.js:15]: I tend to agree, but I attempted to remove it and it broke. I added a comment and it will be in the next commit.

File: package-lock.json (hunk 2 above)
Comment (gcampbell-msft) [reply package-lock.json:16681]: Testing.

Comment (gcampbell-msft) [reply package-lock.json]: I'm attempting to re-run it, but it seems that the macos build fails.

Comment (gcampbell-msft) [reply package-lock.json]: Re-ran, it passed.

**Review by benmcmorran — APPROVED** [review]
